solutions/usaco-1187.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: usaco-1187
+source: USACO Gold 2022 January
+title: Tests for Haybales
+author: Qi Wang
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_prob3_gold_jan22.html)
+
+## Implementation
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int N, time;
```

Comment:
[suggestion] change `time` to another variable name since it's a [builtin function](https://www.cplusplus.com/reference/ctime/time/) although it's fine since you're using it as a variable